.travis.yml

```diff
@@ -5,11 +5,13 @@ php:
 before_script:
 - sudo apt-get update -qq
 - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5 php5-curl php5-mysql php5-intl php5-gd > /dev/null
+- cat /etc/apache2/envvars | grep APACHE_RUN
 - sudo chmod 777 -R /home/travis/build/redCOMPONENT-COM/redCORE/
 - sudo ln -s /home/travis/build/redCOMPONENT-COM/redCORE/tests/ /var/www/tests
 - sudo sed -i -e "s,AllowOverride[ ]None,AllowOverride All,g" /etc/apache2/sites-available/default
 - sudo /etc/init.d/apache2 restart
-- composer install --dev --prefer-dist
+- if [ ! -f composer.phar ]; then curl -sS https://getcomposer.org/installer | php; fi
```

Comment:
great idea!. I wanted to keep the composer.phar in case anyone wants to run the tests outside travis and doesn't have composer in the system as a global bin